app/modules/transcribe/subjects.factory.js

```diff
@@ -7,7 +7,7 @@ require('./transcribe.module.js')
     .factory('SubjectsFactory', SubjectsFactory);
 
 // @ngInject
-function SubjectsFactory($q, localStorageService, zooAPI, zooAPIProject) {
+function SubjectsFactory($q, localStorageService, zooAPI, zooAPIConfig, zooAPIProject) {
```

Comment:
is this new param zooAPIConfig magically set somewhere? If so is order here important or is it like global vars?